doc/users.md

```diff
@@ -1 +1,872 @@
 # Users Module
+Module | HTTP Method | URL | Description 
+--- | --- | --- | ---
+[Add User](#add) | POST | /add | Add User
+[Edit User](#edit) | PUT | /edit | Edit User
+[View User](#view) | GET | /view | View User
+[Delete User](#delete) | DELETE | /delete | Delete User
+
+## <a name="add"></a>Add Data User
+
+### Endpoint 
+POST /add
+
+### Database
+![](./user-layer.png)
+
```

Comment:
- is_login, last_login, last_activity are not required in stateless concepts
- can you explain detail what 'type_akses' field is for?
- if you already have a company_id, you don't need to include company_code and company_name in the users table. Because it's will provide additional responsibility to update the users table while there is an update in the companies table.
- point above also applies to region_code, kios_code, branch_code, and branch_name.
- can you explain detail what 'key' field is for?

Reply:
because the table is stolen from ahass
key field is unused right now

Reply:
remove is_login, last_login, last_activity, company_code, company_name, region_code, kios_code, branch_code, branch_name, and kunci